specification/communication/data-plane/CallAutomation/preview/2024-09-01-preview/communicationservicescallautomation.json

```
@@ -3648,6 +3679,138 @@
         }
       }
     },
+    "CallSessionEndReason": {
+      "description": "Reason call ended",
+      "enum": [
+        "sessionStillOngoing",
+        "callEnded",
+        "initiatorLeft",
+        "handedOverOrTransfered",
+        "maximumSessionTimeReached",
+        "callStartTimeout",
+        "mediaTimeout",
+        "audioStreamFailure",
+        "allInstancesBusy",
+        "teamsTokenConversionFailed",
+        "reportCallStateFailed",
+        "reportCallStateFailedAndSessionMustBeDiscarded",
+        "couldNotRejoinCall",
+        "invalidBotData",
+        "couldNotStart",
+        "appHostedMediaFailureOutcomeWithError",
+        "appHostedMediaFailureOutcomeGracefully",
+        "handedOverDueToMediaTimeout",
+        "handedOverDueToAudioStreamFailure",
+        "speechRecognitionSessionNonRetriableError",
+        "speechRecognitionSessionRetriableErrorMaxRetryCountReached",
+        "handedOverDueToChunkCreationFailure",
+        "chunkCreationFailed",
+        "handedOverDueToProcessingTimeout",
+        "processingTimeout",
+        "transcriptObjectCreationFailed"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "CallSessionEndReason",
+        "modelAsString": true
+      }
+    },
+    "RecordingResultResponse": {
```

Since this will also be the model name in the SDK, let's shorten it to

```suggestion
    "RecordingResult": {
```

---

The current convention for CallAutomation response types is to end them with "Response". I'll stick to the convention for now.